includes/init.php

```diff
@@ -28,13 +38,30 @@ function tsml_single_template($template)
         global $post;
         if ($post->post_type == 'tsml_meeting') {
             $user_theme_file = get_stylesheet_directory() . '/single-meetings.php';
+           
             if (file_exists($user_theme_file)) {
-                return $user_theme_file;
+                 if ($tsml_user_interface == 'tsml_ui') { 
+                    wp_redirect(add_query_arg('tsml_ui', $tsml_user_interface));
```

Comment:
this still isn't working. i did some digging and there are a few things going on here:

* `$tsml_user_interface` needs to be added to `global` above
* this shouldn't be inside the `if (file_exists` statement, because that indicates if a user has supplied their own meeting detail page template, which is immaterial here
* it redirects to the wrong place. it's just adding `?tsml_ui=tsml_ui` to the URL. we want to redirect to the meetings page (`get_post_type_archive_link('tsml_meeting')`) and append `&` or `?` depending on if there's a query string, then `meeting=` and the meeting slug